pkg/agent/testdata/AKSUbuntu1604+Containerd/line70.sh

```diff
@@ -518,6 +535,18 @@ EOF
 #
 iptables -I FORWARD -d 168.63.129.16 -p tcp --dport 80 -j DROP
 iptables -I FORWARD -d 168.63.129.16 -p tcp --dport 32526 -j DROP
+EOF
+
+    primaryNicIP=$(getPrimaryNicIP)
+    ENSURE_IMDS_RESTRICTION_DROP_IN="/etc/systemd/system/kubelet.service.d/10-ensure-imds-restriction.conf"
+    mkdir -p "$(dirname "${ENSURE_IMDS_RESTRICTION_DROP_IN}")"
+    touch "${ENSURE_IMDS_RESTRICTION_DROP_IN}"
+    chmod 0600 "${ENSURE_IMDS_RESTRICTION_DROP_IN}"
+    tee "${ENSURE_IMDS_RESTRICTION_DROP_IN}" > /dev/null <<EOF
```

Comment:
nit: if we use `cat` instead of `tee` then we won't have to worry about piping stdout to `/dev/null`

Reply:
there're several other places using `tee` to write the drop in file, should be OK to follow the convention?

Reply:
that's fine for now, though changing that convention is on my refactor list :)